### PR TITLE
Edits and suggestions.

### PR DESCRIPTION
**Pull request by kevinkle**

### CODE REVIEW

File: paper-webserver.tex [hunk 1]
```diff
@@ -227,8 +235,8 @@ \subsection{Service Virtualization}
 \begin{figure}[!hb]
 \begin{center}
 \includegraphics[width=\textwidth]{images/docker}
-\end{center}
-\caption{The various Docker containers used in Spfy. Arrows represent the connections between different containers, and the entire platform can be recreated with a single command using its Docker-Compose definition.}
+\end{centerbut }
```
Comment (kevinkle) [inline paper-webserver.tex]: ?

File: paper-webserver.tex [hunk 2]
```diff
@@ -273,18 +281,21 @@ \section{DISCUSSION}
...
+The ability to integrate data from different source technologies, merge submissions from other labs, and distribute computations over fault-tolerant systems are now required \cite{schatz2015biological}.
 
-One of the key goals in developing Spfy was to accommodate and store a variety of result formats, and then to make the data from these results retrievable and usable as inputs for downstream analyses, such as predictive biomarker discovery. We have shown how a graph database can accommodate the results from a variety of bioinformatics programs, and how Spfy is performant for data retrieval on the results for multiple analyses among over 10,000 genomes, providing results from big-data comparisons in the same efficiency as old analyses on single files.
+One of the key goals in developing Spfy was to accommodate and store a variety of result formats, and then to make the data from these results retrievable and usable as inputs for downstream analyses, such as predictive biomarker discovery. 
+
+% This has not been shown in this paper; it has only been stated. We should probably have a large-scale analysis for an example, with accompanying times etc.
```
Comment (kevinkle) [inline paper-webserver.tex]: noted!

File: paper-webserver.tex (hunk 1 above)
Comment (chadlaing) [reply paper-webserver.tex]: hmmm, not sure what I did there

Comment (chadlaing) [reply paper-webserver.tex]: It should read 
"The Docker containers used in Spfy. Arrows represent the connections between different containers, and the entire platform can be recreated with a single command using its Docker-Compose definition."

File: tables/database.tex [hunk 3]
```diff
@@ -1,8 +1,10 @@
 % Please add the following required packages to your document preamble:
 % \usepackage{graphicx}
+
+% We need to reference the Tables in the text and explain them. Right now, this table isn't that informative. Eg. what is the importance of the names? Is this a lot of data etc.? If all entries in a column are the same, the column can be removed.
```
Comment (kevinkle) [inline tables/database.tex]: I'm thinking this table can be removed.

Comment (kevinkle) [reply tables/database.tex]: Going to remove this and replace it with ```analysis run-time / throughput with different levels of parallelization
particularly for statistical tests
should do a 1 genome = X number VFs, Y number of genomes for Y*X retrieval/analysis```

File: paper-webserver.tex (hunk 2 above)
Comment (kevinkle) [reply paper-webserver.tex]: I'm going to put the description of the large-scale analysis example into the `Functionality` section (and just mention it in the comparison to BAP). Timings will go in the `Results` section along with other timings.

**Pull request merged**